admin_app/src/app/page.tsx

```diff
@@ -12,6 +12,7 @@ export default function Home() {
   const [cards, setCards] = useState<Content[]>([]);
   const [filteredCards, setFilteredCards] = useState<Content[]>([]);
   const [cardToEdit, setCardToEdit] = useState<Content | null>(null);
+  const [newCardTitle, setNewCardTitle] = useState("");
```

Comment:
We'd want to combine "cardTitle" and "cardText" into a "cardData" object. That will make it a lot cleaner and will make sure we don't forget to update one when we do the other.

Reply:
I think you can draw inspiration from how we do [login](https://github.com/IDinsight/aaq-core/blob/4ff76ed8e70c72eb93a72be4504b261b746e93a9/admin_app/src/app/login/page.tsx#L10C78-L10C78) where we have username and password fields.

Reply:
Sweet that makes sense!